Lib/fontParts/fontshell/point.py

```diff
@@ -11,13 +11,16 @@ def _init(self, wrap=None):
         if wrap is None:
             wrap = self.wrapClass((0, 0))
         super(RPoint, self)._init(wrap=wrap)
-
+        
```

Comment:
Trailing whitespace

Reply:
Yeah, just caught. Perils of doing this quickly via the web editor